pkg/trace/config/features/features.go

```diff
@@ -0,0 +1,47 @@
+package features
```

Comment:
Would be nice to add some package level docs too...
```suggestion
// Package features ...
package features
```